docs/features/software-catalog/well-known-annotations.md

```diff
@@ -41,27 +41,6 @@ expecting a two-item array out of it. The format of the target part is
 type-dependent and could conceivably even be an empty string, but the separator
 colon is always present.
 
-### backstage.io/definition-at-location
```

Comment:
could you move this one down to the bottom of the document for deprecated annotations just to be sure?

Reply:
For sure, done!